packages/circle-react-elements/src/index.tsx

```diff
@@ -20,3 +20,4 @@ export * from './components/WalletReceive';
 export * from './components/WalletSetDetails';
 export * as formats from './lib/format';
 export * as utils from './lib/utils';
+export * from './styles/tailwind.preset';
```

Comment:
notice that preset has its own build step and export settings to not mix up the configuration with usage, see [packages/circle-react-elements/package.json](https://github.com/ChainSafe/web3-circle-libs/blob/de79745a1f790826d65c9cf84acbb337e8226913/packages/circle-react-elements/package.json#L35)